package.json

```
@@ -108,6 +108,7 @@
 		"@php-wasm/node": "^0.9.44",
 		"@php-wasm/scopes": "^0.9.44",
 		"@php-wasm/universal": "^0.9.44",
+		"@reduxjs/toolkit": "^2.5.0",
```

A new [version 2.5.1](https://www.npmjs.com/package/@reduxjs/toolkit?activeTab=versions) was released 3 days ago. We could update it.
```suggestion
		"@reduxjs/toolkit": "^2.5.1",
```